packages/foundry-react-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -423,16 +423,19 @@ const Dropdown = ({
     >
       <Button
         StyledContainer={StyledValueContainer}
-        containerProps={{
-          isOpen,
-        }}
         id={`${name}-button-value`}
         color={defaultedColor}
         onClick={(e: React.MouseEvent) => e.preventDefault()}
         onMouseDown={clickHandler}
         variant={variant}
         containerRef={valueContainerRef}
         {...valueContainerProps}
+        containerProps={{
+          isOpen,
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-expect-error
+          ...(valueContainerProps ? valueContainerProps.containerProps : {}),
```

Comment:
It is worried about valueContainerProps being undefined, so I covered for that possibility and threw an expect error on it

Reply:
I am unsure what the error here would be. If valueContainerProps is undefined, it should choose the second option in the ternary, right?

Reply:
Yep - typescript isn't that smart unfortunately. Of course the "right" option would probably be to improve the typing here - but we're using a generic subcomponent type